doc/man/cockpit.conf.xml

```diff
@@ -196,6 +196,20 @@ ProtocolHeader = X-Forwarded-Proto
     </variablelist>
   </refsect1>
 
+  <refsect1 id="cockpit-conf-session">
+    <title>Session</title>
+    <variablelist>
+      <varlistentry>
+        <term><option>IdleTimeout</option></term>
+        <listitem>
+          <para>Time in minutes after which session expires and user is logged out if no user action
+          has been performed in the given time. Set to <literal>0</literal> to disable session timeout.
+          When not specified, the default is <literal>15</literal> minutes.</para>
```

Comment:
Let's please document the "standard login" behaviour. Maybe:

This idle timeout only applies to interactive password logins. With non-interactive authentication methods like Kerberos, OAuth, or certificate login, the browser cannot forget credentials, and thus automatic logouts are not useful for protecting credentials of forgotten sessions.

Maybe @allisonkarlitskaya has an idea how to tidy the language a bit?